PACKAGE = "homework1"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

integrationEnum =   gen.enum([  gen.const("Euler", int_t, 0, "Euler integration method"),
                    gen.const("RK", int_t, 1, "Runge Kutta integration method")], 
                    "Enumeration to set the integration method used to compute odometry")

gen.add("integrationMethod", int_t, 0, "parameter that represent the integration method thta will be used", 0, 0, 1, edit_method = integrationEnum)

exit(gen.generate(PACKAGE, "robotOdometry", "parameters"))